server/id/id.go

```diff
@@ -119,13 +135,13 @@ func (alloc *allocatorImpl) rebaseLocked() error {
 		return errs.ErrEtcdTxnConflict.FastGenByArgs()
 	}
 
-	log.Info("idAllocator allocates a new id", zap.Uint64("alloc-id", end))
-	idallocGauge.Set(float64(end))
+	log.Info("idAllocator allocates a new id", zap.String("label", alloc.label), zap.Uint64("alloc-id", end))
```

Comment:
The docs of the PD recovery process may need to be updated.

Reply:
Curiously, is there anything broken so we have to update PD recovery process?

Reply:
> Curiously, is there anything broken so we have to update PD recovery process?

It's this block related to finding the maximum allocated ID
https://github.com/pingcap/docs/blob/master/pd-recover.md#get-allocated-id-from-pd-log

Reply:
Once we recover PD, do we also need to recover all id allocators?